src/test/javascript/integration/content_type_default_spec.js

```diff
@@ -27,19 +31,28 @@ describe(__filename, function () {
 
         utils.httpPost(url, headers, body)
             .then(function (response) {
-                expect(response.statusCode).toEqual(201);
-                expect(response.body._links.channel.href).toEqual(channelResource);
-                itemURL = response.body._links.self.href;
+                expect(getProp('statusCode', response)).toEqual(201);
+                const links = fromObjectPath(['body', '_links'], response) || {};
```

Comment:
Out of genuine curiosity, why only go down to the `_links` level and then break it into `channel` and `self`?

Couldn't you do something like this?

    const channelHREF = fromObjectPath(['body', '_links', 'channel', 'href'], response);
    expect(channelHREF).toEqual(channelResource);
    
    itemURL = fromObjectPath(['body', '_links', 'self', 'href'], response);

Reply:
I think I just did it this way because it looked better on the page to me at the time, no reason overly compelling